patches/minecraft/net/minecraft/world/entity/Mob.java.patch

```diff
@@ -32,3 +32,13 @@
           if (entity != null) {
              double d0 = entity.m_20280_(this);
              int i = this.m_6095_().m_20674_().m_21611_();
+@@ -988,7 +_,8 @@
+       } else {
+          this.m_21559_(false);
+       }
+-
++      net.minecraftforge.event.entity.EntityFinalizeSpawnEvent e = new net.minecraftforge.event.entity.EntityFinalizeSpawnEvent(this, p_21435_, p_21436_, p_21437_, p_21438_);
++      net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(e);
```

Comment:
This can just be one line.
`net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.entity.EntityFinalizeSpawnEvent(this, p_21435_, p_21436_, p_21437_, p_21438_));`